trackintel/analysis/tracking_quality.py

```diff
@@ -106,13 +109,19 @@ def temporal_tracking_quality(source, granularity="all", max_iter=60):
         # get the tracked week relative to the first day
         start_date = df["started_at"].min().date()
         df["week"] = df["started_at"].apply(lambda x: (x.date() - start_date).days // 7)
-        dict_week_num = dict(zip(df["week"], df["started_at"].apply(lambda x: pd.to_datetime(x).week)))
+        dict_start_week_num = dict(
+            zip(df["week"], df["started_at"].apply(lambda x: pd.to_datetime(x).isocalendar()[1]))
+        )
+        dict_finish_week_num = dict(
```

Comment:
same as above, I think `dict_start_week_num == dict_finish_week_num`

Reply:
same discussion, will be resolved if we resolve the first [comment](https://github.com/mie-lab/trackintel/pull/328#discussion_r719583534)